invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/Identifiers/PIDField/components/OptionalDOIoptions.js

```diff
@@ -26,6 +26,13 @@ class OptionalDOIoptionsCmp extends Component {
   _render = (cmp, shouldWrapPopup, message) =>
     shouldWrapPopup && message ? <Popup content={message} trigger={cmp} /> : cmp;
 
+  componentDidMount() {
+    const { isManagedSelected, isNoNeedSelected, onManagedUnmanagedChange } =
+      this.props;
+    // When the component is mounted we need to call the callback to sync the state
+    onManagedUnmanagedChange(isManagedSelected, isNoNeedSelected);
```

Comment:
hmmm I don't understand this.... here we call a property passed down from the parent component.... it feels a bit hackish...can you ellaborate on what is the problem please?